src/github.com/getlantern/flashlight/main/main.go

```diff
@@ -305,17 +297,22 @@ func parseFlags() {
 
 // showExistingUi triggers an existing Lantern running on the same system to
 // open a browser to the Lantern start page.
-func showExistingUi(tcpAddr string) {
-	url := "http://" + tcpAddr + "/startup"
+func showExistingUi(addr string) error {
+	url := "http://" + addr + "/startup"
 	log.Debugf("Hitting local URL: %v", url)
 	resp, err := http.Get(url)
 	if err != nil {
 		log.Debugf("Could not hit local lantern")
-		if err = resp.Body.Close(); err != nil {
-			log.Debugf("Error closing body! %s", err)
+		if resp.Body != nil {
```

Comment:
Yeah, I actually got a panic here during development, that's what prompted me to fix it.